lib/TwoFactorAuth.php

```diff
@@ -4,12 +4,12 @@
 
 namespace RobThree\Auth;
 
+use function hash_equals;
```

Comment:
is this necessary or just your IDE being "helpful"?

Reply:
necessary is a big word, but basically when you're in a namespace, the compiled code (opcode) will first try and load the namespaced function, then fallback to the global namespace function.

So to prevent this useless step, there are two options: using `\hash_equals` or using the `use` keyword, and because I find codebase literred with antislash to be bad, I prefer the second way.

Also, the linter currently enforces this with the rule `global_namespace_import`.

Reply:
no worries!